pyproject.toml

```
@@ -7,7 +7,7 @@ name = "joinmarket"
 version = "0.9.12dev"
 description = "Joinmarket client library for Bitcoin coinjoins"
 readme = "README.md"
-requires-python = ">=3.8,<3.12"
+requires-python = ">=3.8,<3.13"
```

I think we can remove upper version limit, unless we encounter problems, as it was before.

```suggestion
requires-python = ">=3.8"
```

---

Just tested install script with Python 3.13.0b1: it fails building wheel for cffi
I suggest to leave it constrain to <3.13 until we manage to have it working.

```
      gcc -fno-strict-overflow -Wsign-compare -DNDEBUG -g -O3 -Wall -fPIC -DFFI_BUILDING=1 -DUSE__THREAD -DHAVE_SYNC_SYNCHRONIZE -I/home/simon/Sources/joinmarket-clientserver/jmvenv/lib/libffi-3.2.1/include -I/home/simon/Sources/joinmarket-clientserver/jmvenv/include -I/home/simon/.pyenv/versions/3.13.0b1/include/python3.13 -c src/c/_cffi_backend.c -o build/temp.linux-x86_64-cpython-313/src/c/_cffi_backend.o
      src/c/_cffi_backend.c: In function ‘b_do_dlopen’:
      src/c/_cffi_backend.c:4532:22: warning: ‘Py_FileSystemDefaultEncoding’ is deprecated [-Wdeprecated-declarations]
       4532 |                      Py_FileSystemDefaultEncoding, &filename_or_null, &flags))
            |                      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
      In file included from /home/simon/.pyenv/versions/3.13.0b1/include/python3.13/Python.h:90,
                       from src/c/_cffi_backend.c:2:
      /home/simon/.pyenv/versions/3.13.0b1/include/python3.13/fileobject.h:22:46: note: declared here
         22 | Py_DEPRECATED(3.12) PyAPI_DATA(const char *) Py_FileSystemDefaultEncoding;
            |                                              ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
      src/c/_cffi_backend.c: In function ‘_my_PyErr_WriteUnraisable’:
      src/c/_cffi_backend.c:6121:9: error: implicit declaration of function ‘_PyErr_WriteUnraisableMsg’; did you mean ‘PyErr_WriteUnraisable’? [-Wimplicit-function-declaration]
       6121 |         _PyErr_WriteUnraisableMsg(PyText_AS_UTF8(s), NULL);
            |         ^~~~~~~~~~~~~~~~~~~~~~~~~
            |         PyErr_WriteUnraisable
      error: command '/usr/bin/gcc' failed with exit code 1
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for cffi
```